Formula/krakend.rb

```diff
@@ -13,7 +13,7 @@ class Krakend < Formula
   end
 
   depends_on "dep" => :build
-  depends_on "go" => :build
+  depends_on "go@1.10" => :build
```

Comment:
This change should be reverted, as the release requires 1.11

Reply:
@alombarte if the new version build with go, we would welcome a pull request